lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -84,6 +89,8 @@ def purchase(money, payment, options = {})
           if payment.is_a?(ApplePayPaymentToken)
             r.process { tokenize_apple_pay_token(payment) }
             payment = StripePaymentToken.new(r.params["token"]) if r.success?
+          elsif ach?(payment)
```

Comment:
I wonder if this state should be moved outside the `MultiResponse`?

Also, do we return a Response-based error in other adapters? Seems like a `Response` should be reserved for when an actual call is made. What other immediate error return states can we use?